### PR TITLE
Refactor AsyncSpringLiquibaseTest

### LINKED ISSUE #2627: Refactor AsyncSpringLiquibaseTest
The current implementation of `AsyncSpringLiquibaseTest` is mainly testing implementation details while using a class that should be deleted (LogbackRecorder). A refactoring might be a good idea here

### PR DESCRIPTION
**Pull request by murdos**
SlownessThreshold of AsyncSpringLiquibase is now configurable, this allows speeding up tests.

Fixes #2627

### CODE REVIEW

**Comment by DamnClin** [comment]
It is better! Can we delete `LogbackRecorder`?

File: src/main/resources/generator/server/springboot/dbmigration/liquibase/test/AsyncSpringLiquibaseTest.java.mustache [hunk 1]
```diff
@@ -292,19 +169,13 @@ class AsyncSpringLiquibaseTest {
...
-          Thread.sleep(sleep);
-        } catch (InterruptedException x) {
-          // This should never happen
-          throw new Error(x);
-        }
+      try {
+        Thread.sleep(fakeDuration.toMillis());
+      } catch (InterruptedException x) {
+        // This should never happen
+        throw new Error(x);
```
Comment (DamnClin) [inline src/main/resources/generator/server/springboot/dbmigration/liquibase/test/AsyncSpringLiquibaseTest.java.mustache:176]: You can throw an AssertionError

**Comment by murdos** [comment]
> It is better! Can we delete `LogbackRecorder`?

Done! I wasn't aware to was only used by this test :smile:

**Review by DamnClin — APPROVED** [review]

**Comment by DamnClin** [comment]
> > It is better! Can we delete `LogbackRecorder`?
> 
> Done! I wasn't aware to was only used by this test smile

Was my first concern with this code :)

**Pull request merged**